meet-data/raw/raw-probe

```diff
@@ -9,10 +9,16 @@ import datetime
 import os
 import shutil
 import sys
-sys.path.append('../')
+sys.path.append('scripts')
```

Comment:
I think this would prevent executing the probe from the federation directory. Maybe we want both "../../scripts" as well as "scripts"? Or the Makefile could set $PYTHONPATH itself?